Makefile

```diff
@@ -360,6 +361,7 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 endif
 
 test-server: test-te test-ee ## Runs tests.
+	find . -type d -name data -not -path './vendor/*' | xargs rm -rf
```

Comment:
I'm not sure about it. Running the unit tests remove any data directory in mattermost-server repo? I'm not sure if it's something safe.

Reply:
After running the tests it creates a bunch of data directories from the tests. This just deletes them.